app/build.gradle

```diff
@@ -23,7 +24,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
         debug {
-            testCoverageEnabled true
+            testCoverageEnabled false
```

Comment:
Oh yes. I actually had to disable it locally because it prevented me from launching the app on the emulator. I don't know why though. I'll re-enable them but I'll need to figure out why it fails locally.